config/i18n.ts

```diff
@@ -118,6 +118,11 @@ const locales: LocaleObjectData[] = [
     file: 'fi-FI.json',
     name: 'Suomea',
   },
+    {
```

Comment:
remove 2 spaces from here (it has 4 spaces), should be 2, you can run `pnpm lint` to check there are no errors